tests/test_dcos_e2e/backends/docker/test_docker.py

```diff
@@ -556,3 +556,24 @@ def test_one_master_host_port_map(self) -> None:
                 }],
             }
             assert master_port_settings == expected_master_port_settings
+
+
+class TestDevices:
```

Comment:
Please add a test for each of the public methods in `DockerLoopbackVolume`.

Please add a test for the `labels` parameter.